modules/webgl/src/classes/program.js

```diff
@@ -70,6 +70,7 @@ export default class Program extends Resource {
     // uniforms
     this.uniforms = {};
 
+    this._texturesUniforms = {};
```

Comment:
Nit: `textureUniforms` (no `s`)...
Nit: Initialize the field to `null` in the constructor. I always feel it is a little dubious to slap on fields dynamically later....
Q: What is the purpose of this new field? Just to hold on to these to prevent GC? Maybe add a comment to explain? Don't we need the same logic for other objects, such as Framebuffer?

Reply:
It's to avoid looping through all uniforms to bind the textures. Deck in particular sets a lot of uniforms, few of which are textures so this was wasteful.